packages/react-vapor/src/components/blankSlate/BlankSlate.tsx

```diff
@@ -8,6 +8,7 @@ import {Svg} from '../svg/Svg';
 export interface IBlankSlateProps extends React.ClassAttributes<BlankSlate> {
     title?: string;
     description?: string;
+    dangerouslySetDescription?: string;

```

Comment:
Why though? 

Could `description` just be of type `React.ReactNode` instead?

Reply:
changed the `description` type to `React.ReactNode` but got this : 

![Screenshot from 2019-09-11 15-52-29](https://user-images.githubusercontent.com/45688129/64729999-53637080-d4ac-11e9-8c89-039437128acb.png)

Is there something else I don't know of I could do to make that work ?

Reply:
When you use the prop you'll need to send 
```typescript
<BlankSlate
    title="title test"
    description={<p dangerouslySetInnerHTML={{__html: "description test <a href='https://www.google.com'>the link</a>"}} />
/>
```

Reply:
Oh I see. So we still have to use the `DangerouslySetInnerHTML` in both case. If both of you think that it is better this way than adding another props lets do that. Thank you guys for the `React.ReactNode` type, I will have to read a bit more documentation about it !

Reply:
is it ok if I just skipped the `dangerouslySetThing` ?
`<div className="form-group">
                    <label className="form-control-label">BlankSlate with title and description with a link</label>
                    <BlankSlate title="title test" description={<p>this is a description with a link to <a href='https//www.google.com'>this website</a></p>} />
                </div>`
![Screenshot from 2019-09-11 20-03-54](https://user-images.githubusercontent.com/45688129/64743804-52900600-d4cf-11e9-9661-9732e9f65cae.png)
:1st_place_medal: